src/framework/uicomponents/qml/MuseScore/UiComponents/internal/InfoPanel.qml

```diff
@@ -0,0 +1,191 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+
+import MuseScore.Ui 1.0
+import MuseScore.UiComponents 1.0
+
+PopupPanel {
+    id: root
+
+    property string title: ""
+    property var additionalInfoModel: undefined
+    property string description: ""
+
+    property Component buttonsPanel: null
+    property var mainButton: null
+
+    property var currentOperationButton: undefined
+
+    property NavigationPanel contentNavigation: NavigationPanel {
+        name: root.objectName != "" ? root.objectName : "InfoPanel"
+
+        enabled: root.visible
+        section: root.navigationSection
+        order: 1
+
+        onActiveChanged: function(active) {
+            if (active) {
+                root.forceActiveFocus()
+            }
+        }
+    }
+
+    height: 360
+
+    visible: false
+
+    accessible.name: root.title
+
+    function setProgress(status, indeterminate, current, total) {
```

Comment:
It seems that these functions (setProgress and resetProgress) are no longer needed